proto/protovalidate/buf/validate/validate.proto

```diff
@@ -330,10 +328,24 @@ enum Ignore {
   //   optional int32 value = 1 [
   //     default = -42,
   //     (buf.validate.field).int32.gt = 0,
-  //     (buf.validate.field).ignore = IGNORE_DEFAULT
+  //     (buf.validate.field).ignore = IGNORE_IF_DEFAULT_VALUE
   //   ];
   // }
-  IGNORE_DEFAULT = 2;
+  IGNORE_IF_DEFAULT_VALUE = 2;
+  // Deprecated: Use IGNORE_IF_DEFAULT_VALUE
+  IGNORE_DEFAULT = 2 [deprecated = true];
+
+  // The validation rules of this field will not be evaluated.
```

Comment:
Perhaps add a sentence such as "this is useful for situations where you want to temporarily turn off a rule, or [insert other reasons this is useful". Perhaps also add another sentence "this replaces `skipped` from protoc-gen-validate".

Reply:
Cool with the first, I don't really want to document this based off its predecessor however.